test/e2e/config.yaml

```diff
@@ -31,8 +31,8 @@ chains:
       name: archway
       chain_id: archway-1
       image:
-        repository: archwaynetwork/archwayd
-        version: sha-8f53ac8
```

Comment:
Why not use this archway image that is on dockerhub?

Reply:
I am getting no such image response. are you able to pull?

Reply:
Yeah works for me
https://hub.docker.com/layers/archwaynetwork/archwayd/sha-8f53ac8/images/sha256-533fa41e3e39c9b4af1a49baffd6e7db782c2e65da9be7b1d105f2f6f643475d?context=explore

Reply:
I guess, https://hub.docker.com/r/archwaynetwork/archwayd is official docker image

Reply:
Oh I see, its platform related then.
I can pull using amd64 but not arm64.